src/main/java/uk/gov/hmcts/reform/sandl/snlrules/security/S2SAuthenticationConfig.java

```diff
@@ -18,6 +18,7 @@
 @Setter
 class S2SAuthenticationConfig {
     JwtCredentials rules;
+    boolean enabled;
```

Comment:
what is this for? why there are changes to authentication needed here?

Reply:
We already have that field in application.yaml but haven't used it at all, but it's quite useful during local tests to disable s2s.
Your call whether we want it or not.
Have a look at src/main/java/uk/gov/hmcts/reform/sandl/snlrules/security/S2SAuthenticationService.isDisabled() and src/main/java/uk/gov/hmcts/reform/sandl/snlrules/security/S2SJwtAuthenticationFilter.java. shouldDoTokenCheck(...) to see an example usage.

Reply:
ok